src/components/SearchBar/index.tsx

```diff
@@ -77,13 +105,13 @@ const SearchBar: React.FC<SearchBarProps> = ({
             </IconWithTooltip>
           )
         }
-        onChange={(e) => setInput(e.target.value)}
+        onChange={(e) => handleOnChange(e.target.value)}
         onKeyPress={(e: KeyboardEvent<HTMLInputElement>) =>
           e.key === "Enter" && handleOnSubmit()
         }
         placeholder="optional, regexp to search"
         spellCheck={false}
-        type="search"
+        type={SearchBarActions.Search}
```

Comment:
I don't think we should type this with our enum since it's a native property of `input`.

If you really want you could probably type it using `import { TextInputType } from "@leafygreen-ui/text-input"` but that might be a little overkill

Reply:
Ooops that def wasn't intentional